bot/bot.py

```diff
@@ -306,11 +306,6 @@ async def set_chat_mode_handle(update: Update, context: CallbackContext):
     db.set_user_attribute(user_id, "current_chat_mode", chat_mode)
     db.start_new_dialog(user_id)
 
-    await query.edit_message_text(
```

Comment:
I think it would be better to keep this while changing the next message with `update.effective_message.reply_text`. This will end up with two messages, i.e., confirmation of mode change and welcome.

Reply:
Sorry, don't get it. Can you explain more?

Reply:
Sure, make sense.